### PR TITLE
README update

### PR DESCRIPTION
**Pull request by zurk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,110 +1,113 @@
...
+ 
+This project is the foundation for machine learning on source code (or simply [MLonCode](https://github.com/src-d/awesome-machine-learning-on-source-code)) research and development. It abstracts machine learning engineer daily routine like features extraction, datasets collection and models training, allowing you to focus on higher level tasks.
 
-This project is the foundation for [MLonCode](https://github.com/src-d/awesome-machine-learning-on-source-code) research and development. It abstracts feature extraction and training models, allowing you to focus on higher level tasks.
+Source{d} ml is tightly coupled with [Engine](https://docs.sourced.tech/engine) and delegates all the feature extraction parallelization to it. It is written in Python3 and has been tested on Linux and macOS.
```
Comment (vmarkovtsev) [inline README.md]: source{d}, we never capitalize

File: doc/cmd/preprocrepos.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Preprocrepos command
+
+This command allows you to preprocess your data before passing it to any command you need.
+
+## Preprocrepos pipeline:
```
Comment (eiso) [inline doc/cmd/preprocrepos.md]: This section needs command line examples for each point you mention below

Comment (zurk) [reply doc/cmd/preprocrepos.md]: They will be in a separate file. 
I will add a links here

**Comment by zurk** [comment]
@eiso 
I add examples I think that on the current stage my work is done. 
I tried to cover most important aspects. Will reread it tomorrow morning with a fresh head.
Add more links and fix existing ones. 

Fill free to tell me if you need something more.

**Pull request merged**